chart/templates/openshift-gitops/subscription.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-{{ if (index .Values "openshift-gitops") }}
+{{ if and (index .Values "openshift-gitops") (index .Values "openshift-gitops" "enabled") }}
```

Comment:
Same as other question about the structure of the conditional

Reply:
See comment https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7#discussion_r1779167022